docs/rule/kotlin_test.soy

```diff
@@ -0,0 +1,118 @@
+{namespace kotlin_test}
+
+/***/
+{template .soyweb}
+  {call buck.page}
+    {param title: 'kotlin_test()' /}
+    {param navid: 'rule_kotlin_test' /}
+    {param prettify: true /}
+    {param description}
+      A rule that is used to define a set of kotlin files that contain tests.
+    {/param}
+    {param content}
+
+{call buck.rule}
+{param status: 'FROZEN' /}
```

Comment:
We don't usually mark new rules as frozen since we might be changing them as we flesh them out.